### ISSUE TITLE
latest docker image built incorrectly, no vnu binary

### ISSUE THREAD

**Issue by piotrleszczynski** [validator/validator#2022]
### Summary

Docker `latest` image doesn't work:
```
> docker run --rm ghcr.io/validator/validator:latest vnu --version
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: exec: "vnu": executable file not found in $PATH
```
There is no `vnu` binary inside container, no `vnu-runtime-image` dir either.

Last tagged image works as expected:
```
> docker run --rm ghcr.io/validator/validator:24.10.17 vnu --version
Picked up JAVA_TOOL_OPTIONS: 
24.10.31
```

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
n/a
```

### Checker error messages

```shell
bash: VNU: command not found
```

### Steps to reproduce

1. Run `docker run --rm ghcr.io/validator/validator:24.10.17 vnu --version`

### Expected behavior

Show version number

### Actual behavior

docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: exec: "vnu": executable file not found in $PATH

**Comment by sideshowbarker** [comment]
Thanks for catching this, and reporting it.

I’ve fixed it, and pushed a new image. Please give it a try — and if it doesn’t work, lemme know.
